app/frontend/src/components/constants.ts

```diff
@@ -1,5 +1,10 @@
 export const CANCEL_UPLOAD = "Cancel upload";
 export const CONFIRM_UPLOAD = "Confirm upload";
+export const COOKIE_MESSAGE = [
+  "We use cookies to ensure that we give you the best experience on our website. If you continue to use this site, we will assume that you are happy with it. You can ",
+  "read more about our terms of service",
```

Comment:
```suggestion
  "read more about our Terms of Service",
```
I think they're meant to be capitalised?